.pipelines/templates/c-api-cpu.yml

```diff
@@ -0,0 +1,269 @@
+
+- name: AdditionalBuildFlags
+  displayName: Additional build flags for build.py
+  type: string
+  default: ''
+
+- name: AdditionalWinBuildFlags
+  displayName: Additional build flags that just for Windows Builds
+  type: string
+  default: ''
+
+- name: OrtNugetPackageId
+  displayName: Package name for nuget
+  type: string
+  default: 'Microsoft.ML.OnnxRuntime.Extensions'
+
+- name: BuildVariant
+  type: string
+  default: 'default'
+
+stages:
+# MacOS_C_API_Packaging_CPU
+- template: c-api-osx-cpu.yml
+  parameters:
+    IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+    AllowReleasedOpsetOnly: 1
+    BuildForAllArchs: true
+
+- stage: Linux_C_API_Packaging_CPU
+  dependsOn: []
+  jobs:
+  - template: c-api-linux-cpu.yml
+    parameters:
+      AdditionalBuildFlags: ${{ parameters.AdditionalBuildFlags }}
+      OrtExtensionsArch: 'x64'
+      PoolName: 'aiinfra-Linux-CPU'
+      IsReleaseBuild: ${{parameters.IsReleaseBuild}}
+  - template: c-api-linux-cpu.yml
+    parameters:
+      AdditionalBuildFlags: ${{ parameters.AdditionalBuildFlags }}
+      OrtExtensionsArch: 'aarch64'
+      PoolName: 'aiinfra-linux-ARM64-CPU-2019'
+      IsReleaseBuild: ${{parameters.IsReleaseBuild}}
+
+- stage: Windows_C_API_Packaging_CPU
+  dependsOn: []
+  jobs:
+  - template: c-api-windows-cpu.yml
+    parameters:
+      DoCompliance: ${{ parameters.DoCompliance }}
+      DoEsrp: ${{ parameters.DoEsrp }}
+      stage_name_suffix: CPU_x86_${{ parameters.BuildVariant }}
+      buildArch: x86
+      msbuildPlatform: Win32
+      packageName: x86
+
+  - template: c-api-windows-cpu.yml
+    parameters:
+      DoCompliance: ${{ parameters.DoCompliance }}
+      DoEsrp: ${{ parameters.DoEsrp }}
+      stage_name_suffix: CPU_arm_${{ parameters.BuildVariant }}
+      buildArch: x64
+      msbuildPlatform: arm
+      packageName: arm
+
+  - template: c-api-windows-cpu.yml
+    parameters:
+      DoCompliance: ${{ parameters.DoCompliance }}
+      DoEsrp: ${{ parameters.DoEsrp }}
+      stage_name_suffix: CPU_arm64_${{ parameters.BuildVariant }}
+      buildArch: x64
+      msbuildPlatform: arm64
+      packageName: arm64
+
+  - template: c-api-windows-cpu.yml
+    parameters:
+      DoCompliance: ${{ parameters.DoCompliance }}
+      DoEsrp: ${{ parameters.DoEsrp }}
+      stage_name_suffix: CPU_x64_${{ parameters.BuildVariant }}
+      buildArch: x64
+      msbuildPlatform: x64
+      packageName: x64
+#Android_Java_API_AAR_Packaging_Full
+- template: c-api-android-cpu.yml
+  parameters:
+    IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+
+- stage: iOS_Full_xcframework
+  dependsOn: []
+  jobs:
+    - template: c-api-ios-cpu.yml
+      parameters:
+        IsReleaseBuild: ${{ parameters.IsReleaseBuild }}
+
+#- stage: Android_Java_API_AAR_Packaging_Full
+#  dependsOn: []
+#  jobs:
+#  - template: ../android_packaging.yml
+#
+#- stage: iOS_Full_xcframework
+#  dependsOn: []
+#  jobs:
+#    - template: ../ios_packaging.yml
+
+
+- stage: NuGet_Packaging_CPU
```

Comment:
If this config is responsible for assembling the nuget package for multiple different platforms, 'c-api-cpu.yml' doesn't really convey that. Can we rename to something more meaningful?